### PR TITLE
Task#7 Feat: Add popover for filters

### LINKED ISSUE #49: Feature: Show all products without any filter first
- [x] #50
> As a user, in the app I can see a list of all products  without any filter first. When I just loaded into the interface, a list of all products will appear so I can see pictures, offers, product names and their prices.

### LINKED ISSUE #50: User story#1: Products: UI Design
- Task#2  Create filter interface and logo icon for manufacturer.

- [x] Full 3 filters

- [x] List logo for manufacturer

- Task#3 Create card details tags for each product

- [x] Must have image at the top
- [x] Label for screen pixel size and type
- [x] Price of the product

- Task#4 Edit the style for the interface

- [x] User can hover the filter button
- [x] Users can hover the logo manufacturer.

- Task#5 Testing the features and display 

- [x] Check if there are enough products

**_Task2, Task3, task4, task 5 One in Task#2 Create UI for products - cards #51_**

- [x] #52
- [x] #58
- [x] #61

### LINKED ISSUE #58: Task#7 Dev Task - Add popover for filters
- #49 

## OVERVIEW
- [x] Add popover for filter
- [x] Add eventlistener
### Design
- Price
image
- Manufacture
image
- Total
image

### PR DESCRIPTION
**Pull request by TheHienDuong**
- Closes #58 
Reference links: https://www.thegioididong.com/dtdd
## DEMO

https://github.com/Agility-Internship/Duong-The-Hien-Javascript/assets/113186169/05ffbbec-2885-4b1f-9290-9c27405b5dd4

### CODE REVIEW

**Comment by TheHienDuong** [comment]
In the latest comment i updated the popover code to html instead of using innerHTML

File: practice-02/src/index.html [hunk 1]
```diff
@@ -17,18 +17,115 @@
...
+                <!-- Filter Total -->
+                <div class="filter-item filter-total">
                     <div class="filter-item__title">Filter</div>
+                    <div class="filter-show show-total">
+                        <div class="show-total-main">
+                            <div class="show-total-item clearfix">
+                                <div class="show-total-title">Manufacture</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30"
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: Miss alt. Please update ALL similar CASES

File: practice-02/src/index.html [hunk 2]
```diff
@@ -17,18 +17,115 @@
...
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="show-total-item">
+                                <div class="show-total-title">Price</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item"> 2 - 4 million </a>
+                                    <a href="#" class="logo-item"> 4 - 8 million </a>
+                                    <a href="#" class="logo-item"> 8 - 15 million </a>
+                                    <a href="#" class="logo-item"> over 15 million </a>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: always uppercase first character. Please check and update all the same cases

File: practice-02/src/index.html [hunk 3]
```diff
@@ -17,18 +17,115 @@
...
-                <div class="filter-total">
+                <!-- Filter Total -->
+                <div class="filter-item filter-total">
                     <div class="filter-item__title">Filter</div>
+                    <div class="filter-show show-total">
+                        <div class="show-total-main">
+                            <div class="show-total-item clearfix">
+                                <div class="show-total-title">Manufacture</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item">
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: If its not a link, Please dont make it's a link, using another HTML tag

Comment (phattruong-agilityio) [reply practice-02/src/index.html]: Update all similar cases

File: practice-02/src/index.html [hunk 4]
```diff
@@ -17,18 +17,115 @@
...
+                                    </a>
+                                    <a href="#" class="logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Mobell42-b_19.jpg" width="30" class="logo" />
+                                    </a>
+                                </div>
+                            </div>
+                            <div class="show-total-item">
+                                <div class="show-total-title">Price</div>
+                                <div class="filter-list ">
+                                    <a href="#" class="logo-item"> 2 - 4 million </a>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: why have space` 2 - 4 million `

File: practice-02/styles/index.css [hunk 5]
```diff
@@ -180,3 +180,116 @@ body {
...
+}
+
+.show-total-main {
+  display: flex;
+  flex-wrap: wrap;
+  height: auto;
+  padding: 15px 0 10px;
+}
+
+.show-total .show-total-item {
```
Comment (phattruong-agilityio) [inline practice-02/styles/index.css:269]: Please using flex for layout

File: practice-02/styles/index.css [hunk 6]
```diff
@@ -180,3 +180,116 @@ body {
...
+  float: left;
```
Comment (phattruong-agilityio) [inline practice-02/styles/index.css]: Dont use float, it's flexbox time

File: practice-02/src/index.html (hunk 1 above)
Comment (TheHienDuong) [reply practice-02/src/index.html]: Yes, I have updated the alt for all image cases

File: practice-02/src/index.html (hunk 2 above)
Comment (TheHienDuong) [reply practice-02/src/index.html]: Yes, I Changed the first letter of the character ( over ... )

File: practice-02/src/index.html (hunk 3 above)
Comment (TheHienDuong) [reply practice-02/src/index.html]: I updated a link into button

File: practice-02/src/index.html (hunk 4 above)
Comment (TheHienDuong) [reply practice-02/src/index.html]: In the sample site I see spaces between them, but I have updated the space between the text prices

File: practice-02/styles/index.css (hunk 5 above)
Comment (TheHienDuong) [reply practice-02/styles/index.css:269]: Yeah, I Fixed

File: practice-02/styles/index.css (hunk 6 above)
Comment (TheHienDuong) [reply practice-02/styles/index.css]: Yeah, I Fixed

File: practice-02/src/index.html (hunk 4 above)
Comment (phattruong-agilityio) [reply practice-02/src/index.html]: We don't have any `the same site` here. Please update,

File: practice-02/src/index.html [hunk 7]
```diff
@@ -17,53 +17,146 @@
...
-                        <img src="//cdn.tgdd.vn/Brand/1/Masstel42-b_0.png" width="30" class="logo" />
-                    </a>
-                    <a href="#" class="logo-item">
-                        <img src="//cdn.tgdd.vn/Brand/1/Itel42-b_54.jpg" width="30" class="logo" />
-                    </a>
-                    <a href="#" class="logo-item">
-                        <img src="//cdn.tgdd.vn/Brand/1/Mobell42-b_19.jpg" width="30" class="logo" />
-                    </a>
+                  <button class="btn logo-item">
+                    <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30" class="logo" alt="Image of the logo of the phone company iphone"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: wrong alt: this is for iphone right? so alt should be iphone

File: practice-02/src/index.html [hunk 8]
```diff
@@ -17,53 +17,146 @@
...
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/samsungnew-220x48-1.png" width="30" class="logo" alt="Image of the logo of the phone company samsung"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: wrong alt: this is for samsung right? so alt should be samsung

File: practice-02/src/index.html [hunk 9]
```diff
@@ -17,53 +17,146 @@
...
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/OPPO42-b_5.jpg" width="30" class="logo" alt="Image of the logo of the phone company oppo"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 10]
```diff
@@ -17,53 +17,146 @@
...
+                    </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/logo-xiaomi-220x48-5.png" width="30" class="logo" alt="Image of the logo of the phone company xiaomi"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 11]
```diff
@@ -17,53 +17,146 @@
...
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/vivo-logo-220-220x48-3.png" width="30" class="logo" alt="Image of the logo of the phone company vivo"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 12]
```diff
@@ -17,53 +17,146 @@
...
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Realme42-b_37.png" width="30" class="logo" alt="Image of the logo of the phone company realme"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 13]
```diff
@@ -17,53 +17,146 @@
...
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Nokia42-b_21.jpg" width="30" class="logo" alt="Image of the logo of the phone company nokia"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 14]
```diff
@@ -17,53 +17,146 @@
...
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Masstel42-b_0.png" width="30" class="logo" alt="Image of the logo of the phone company masstel"/>
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Itel42-b_54.jpg" width="30" class="logo" alt="Image of the logo of the phone company itel"/>
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Mobell42-b_19.jpg" width="30" class="logo" alt="Image of the logo of the phone company mobell"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 15]
```diff
@@ -17,53 +17,146 @@
...
+                      <img src="//cdn.tgdd.vn/Brand/1/Realme42-b_37.png" width="30" class="logo" alt="Image of the logo of the phone company realme"/>
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Nokia42-b_21.jpg" width="30" class="logo" alt="Image of the logo of the phone company nokia"/>
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Masstel42-b_0.png" width="30" class="logo" alt="Image of the logo of the phone company masstel"/>
+                  </button>
+                  <button class="btn logo-item">
+                      <img src="//cdn.tgdd.vn/Brand/1/Itel42-b_54.jpg" width="30" class="logo" alt="Image of the logo of the phone company itel"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 16]
```diff
@@ -17,53 +17,146 @@
...
+                <!-- Filter Total -->
+                <div class="filter-item filter-total">
                     <div class="filter-item__title">Filter</div>
+                    <div class="filter-show show-total">
+                        <div class="show-total-main">
+                            <div class="show-total-item clearfix">
+                                <div class="show-total-title">Manufacture</div>
+                                <div class="filter-list ">
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30" class="logo" alt="Image of the logo of the phone company iphone"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 17]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/samsungnew-220x48-1.png" width="30" class="logo" alt="Image of the logo of the phone company samsung"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 18]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/OPPO42-b_5.jpg" width="30" class="logo" alt="Image of the logo of the phone company oppo"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 19]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/logo-xiaomi-220x48-5.png" width="30" class="logo" alt="Image of the logo of the phone company xiaomi"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 20]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/vivo-logo-220-220x48-3.png" width="30" class="logo" alt="Image of the logo of the phone company vivo"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 21]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Realme42-b_37.png" width="30" class="logo" alt="Image of the logo of the phone company realme"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 22]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Nokia42-b_21.jpg" width="30" class="logo" alt="Image of the logo of the phone company nokia"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 23]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Masstel42-b_0.png" width="30" class="logo" alt="Image of the logo of the phone company masstel"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 24]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Itel42-b_54.jpg" width="30" class="logo" alt="Image of the logo of the phone company itel"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 25]
```diff
@@ -17,53 +17,146 @@
...
+                                    </button>
+                                    <button class="btn logo-item">
+                                        <img src="//cdn.tgdd.vn/Brand/1/Mobell42-b_19.jpg" width="30" class="logo" alt="Image of the logo of the phone company mobell"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

File: practice-02/src/index.html [hunk 26]
```diff
@@ -17,53 +17,146 @@
...
-                <div class="filter-item">
+                <!-- Filter manufacture -->
+                <div class="filter-item manufacture">
                     <div class="filter-item__title">
                         <span class="title">Manufacturer</span>
                     </div>
+                    <div class="filter-show">
+                      <div class="filter-list ">
+                        <button class="btn logo-item">
+                            <img src="//cdn.tgdd.vn/Brand/1/logo-iphone-220x48.png" width="30" class="logo" alt="Image of the logo of the phone company iphone"/>
```
Comment (phattruong-agilityio) [inline practice-02/src/index.html]: update this one

**Comment by phattruong-agilityio** [comment]
Please check comment and update all alt image

**Pull request merged**